activerecord/lib/active_record/persistence.rb

```diff
@@ -286,21 +286,9 @@ def update_column(name, value)
       update_columns(name => value)
     end
 
-    # Updates the attributes directly in the database issuing an UPDATE SQL
-    # statement and sets them in the receiver:
-    #
-    #   user.update_columns(last_request_at: Time.current)
-    #
-    # This is the fastest way to update attributes because it goes straight to
-    # the database, but take into account that in consequence the regular update
-    # procedures are totally bypassed. In particular:
-    #
-    # * \Validations are skipped.
-    # * \Callbacks are skipped.
-    # * +updated_at+/+updated_on+ are not updated.
```

Comment:
I think we should keep most of this, but add a note to this list that reads something along the lines of "attributes are serialized using the same rules as update_all"